prime-router/src/main/kotlin/azure/SendFunction.kt

```diff
@@ -87,28 +87,27 @@ class SendFunction(private val workflowEngine: WorkflowEngine = WorkflowEngine()
                         nextRetryItems += nextRetry
                     }
                 }
-                context.logger.info("For $inputReportId:  finished send().  Calling handleRetry.")
+                logger.info("For $inputReportId:  finished send().  Checking to see if a retry is needed.")
                 handleRetry(
                     nextRetryItems,
                     inputReportId,
                     serviceName,
                     retryToken,
-                    context,
                     actionHistory,
                     event.isEmptyBatch
                 )
             }
         } catch (t: Throwable) {
             // For debugging and auditing purposes
-            val msg = "Send function unrecoverable exception for event: $message"
-            context.logger.log(Level.SEVERE, msg, t)
+            val msg = "Send function unrecoverable exception for event. Mo intervention required: $message"
             actionHistory.setActionType(TaskAction.send_error)
             actionHistory.trackActionResult(msg)
+            logger.error("$ALERT_TRIGGER: ${actionHistory.action.actionResult}", t)
```

Comment:
SendFunction has failed with an unrecoverable exception.  So we use the super secret magic word in a log message.